packages/aws-rfdk/lib/deadline/test/worker-fleet.test.ts

```diff
@@ -1943,3 +1944,71 @@ test('worker fleet does not signal when zero minCapacity', () => {
   expect(fleet.node.metadataEntry[1].type).toMatch(ArtifactMetadataEntryType.WARN);
   expect(fleet.node.metadataEntry[1].data).toMatch(/Deploying with 0 minimum capacity./);
 });
+
+describe('secrets management enabled', () => {
+  let props: WorkerInstanceFleetProps;
+
+  // GIVEN
+  beforeEach(() => {
+    app = new App();
+    stack = new Stack(app, 'Stack');
+    vpc = new Vpc(stack, 'VPC');
+    rcsImage = ContainerImage.fromAsset(__dirname);
+    const version = new VersionQuery(stack, 'VersionQuery');
+    renderQueue = new RenderQueue(stack, 'RQ', {
+      vpc,
+      images: { remoteConnectionServer: rcsImage },
+      repository: new Repository(stack, 'Repository', {
+        vpc,
+        version,
+      }),
+      version,
+    });
+    wfstack = new Stack(app, 'workerFleetStack');
+    props = {
+      renderQueue,
+      vpc,
+      workerMachineImage: new GenericWindowsImage({
```

Comment:
```suggestion
      workerMachineImage: new GenericWindowsImage(),
```

We can remove the empty props `{}` here

Reply:
This is actually the AMI region map and its required. I've collapsed it onto a single line though which makes it looks a bit nicer

Reply:
Ah, I should've double checked the API. Thanks